clippy_lints/src/no_effect.rs

```diff
@@ -40,12 +40,22 @@ declare_lint! {
     "outer expressions with no effect"
 }
 
+/// Check whether this type implements Drop.
+fn has_drop(cx: &LateContext, expr: &Expr) -> bool {
```

Comment:
Yeah, please put this somewhere in `utils`

Reply:
Sure.